### DISCUSSION TITLE
Mobius arithemetics derivation

### DISCUSSION THREAD

**Discussion by SomeoneSerge** [geoopt/geoopt#178] [General]
I'm too lazy and busy to actually check out, but apparently Mobius addition should do something like
\[ P \oplus Q = \Exp_Q(Transport_{O\to Q}(OP)) \]
```
Input: points P, Q; O -- Origin.
OP: Tangent_O = Log_O (P)
QR: Tangent_Q = Transport OP from O to Q
R = Exp_Q (QR)
Output: R
```
It might be useful to see if this (or anything resembling) is true, and then write **tests**
comparing Mobius arithmetic with log/transport/exp stuff bound to $0$

* * *

Otherwise, we got to find time and just find out where's all this stuff derived from

**Comment by ferrine** [comment]
Better pick some time to investigate this. I, to be honest, did not have enough time to check that.
